tests/test_astar.py

```diff
@@ -0,0 +1,118 @@
+import sys
+sys.path.append('../')
+sys.path.append('../../')
+
+from two_d_nav.elements import VelRobot, Obstacle, create_maze
+from two_d_nav.engine import NavigationEngine
+from search.state import State
+from search.location import Location
+from search.a_star import Agent
+import numpy as np
+import constants as c
+
+def define_walls(board, lines):
+  for l in lines:
+    x1 = l[1][0]
+    y1 = l[1][1]
+    x2 = l[2][0]
+    y2 = l[2][1]
+    thickness = l[3]
+    isHorizontal = y1 == y2
+    if isHorizontal:
+      for i in range(thickness):
+        xval = min(x1, x2)
+        for x in range(abs(x1 - x2)):
+          board[y1+i-1][xval+x-1] = c.INFINITY
+    else:
+      for i in range(thickness):
+        yval = min(y1, y2)
+        for y in range(abs(y1 - y2)):
+          board[yval+y-1][x1+i-1] = c.INFINITY
+
+  return board
+
+def define_obstables(board, obs_list):
+  size = c.OBSTACLE_SIZE
+  for o in obs_list:
+    x = int(o.init_x)
+    y = int(o.init_y)
+    for i in range(x, x+size):
+      for j in range(y, y+size):
+        board[j-1][i-1] = c.INFINITY
+  return board
+
+
+
+def create_board(lines, obs_list):
+  board_size = c.BOARD_SIZE
+  board = np.zeros(shape = (board_size, board_size))
+  board = define_walls(board, lines)
+  board = define_obstables(board, obs_list)
+  return board, board_size, board_size
+
+def deconstructPath(path, row, col):
```

Comment:
code style issue, the function name should be snake case.

Reply:
resolved